src/build-time-render/BuildTimeRender.ts

```diff
@@ -27,6 +27,7 @@ const createHash = require('webpack/lib/util/createHash');
 import { parse, HTMLElement } from 'node-html-parser';
 
 import { read as readCache, write as writeCache, remove as removeFromCache, Cache } from './cache';
+import { FeatureMap } from '../static-build-loader/getFeatures';
```

Comment:
@matt-gadd do you want to remove the typings from here too?

Reply:
yes please, i don't like the cross loader imports. the type is wrong for `FeatureMap` anyway I think?